### PR TITLE
Support END in "initial_from_file_timestep"

### LINKED ISSUE #5748: initial_from_file_timestep should be able to grab "end"
If end(ie the last timestep) is tried in the restart system, the following error is thrown.

`ERROR: Input value for variable GlobalParams/initial_from_file_timestep is of the wrong type.
       value = end expected type = i`

Would this be easy to change?

### PR DESCRIPTION
**Pull request by permcody**
closes #5748

### CODE REVIEW

File: framework/src/actions/CopyNodalVarsAction.C [hunk 1]
```diff
@@ -23,7 +23,7 @@ template<>
 InputParameters validParams<CopyNodalVarsAction>()
 {
   InputParameters params = validParams<Action>();
-  params.addParam<int>("initial_from_file_timestep", 2, "Gives the timestep for which to read a solution from a file for a given variable");
+  params.addParam<std::string>("initial_from_file_timestep", "LAST", "Gives the timestep for which to read a solution from a file for a given variable");
```
Comment (dschwen) [inline framework/src/actions/CopyNodalVarsAction.C]: Defaults to "LAST", but the check above is "END"?

File: framework/include/base/SystemBase.h [hunk 2]
```diff
@@ -568,24 +568,39 @@ class SystemTempl : public SystemBase
...
     bool did_copy = false;
     for (std::vector<VarCopyInfo>::iterator it = _var_to_copy.begin();
         it != _var_to_copy.end();
         ++it)
     {
-      did_copy = true;
       VarCopyInfo & vci = *it;
+      int timestep = -1;
+
+      if (vci._timestep == "END")
```
Comment (dschwen) [inline framework/include/base/SystemBase.h]: Default below is "LAST"

Comment (permcody) [inline framework/include/base/SystemBase.h]: Good catch. I changed my mind what to call it and the input file didn't use
the default.

**Comment by jwpeterson** [comment]
It looks like `ExodusII_IO` is 1-based, i.e. if `io.get_num_time_steps();` returns 10, it is valid to call `io.copy_nodal_solution(a, b, c, 10);`

It might be nice to document the `initial_from_file_timestep` parameter to make this more clear, but since it hasn't seemed to confuse anyone yet, I guess it's not a big deal.

**Pull request merged**